specs/AppNotifications/AppNotificationBuider/Headers-spec.md

```diff
@@ -0,0 +1,67 @@
+# Headers
+
+Headers let developers identify related notifications so they can be grouped together in Notification Center.
+![AppNotification Header Example](toast-content-headers.png)
+
+Developers can specify a Header when defining a notification in raw xml, but the AppNotificationBuilder doesn't support it.
+
+```xml
+<toast>
+    <header 
+        id='6289' 
+        title='Camping!!' 
+        arguments='action=openConversation;id=6289'/>
+    <visual>
+        <binding template='ToastGeneric'>
+            <text>Anyone have a sleeping bag I can borrow?</text>
+        </binding>
+    </visual>
+</toast>
+```
+
+Adding support for Headers to the AppNotificationBuilder is straightforward. All is needed is a new Header runtime class to represent the Header.
+
+```idl
+[contract(AppNotificationBuilderContract, 2)]
+runtimeclass AppNotificationHeader
+{
+    AppNotificationHeader();
+    AppNotificationHeader(String id, String title);
```

Comment:
Can the id be non-numeric?

Reply:
This is what the doc says: A developer-created identifier that uniquely identifies this header. If two notifications have the same header id, they will be displayed underneath the same header in Action Center.

So to my it means it can be anything.